docs/docs/guides/integrations/notdiamond.md

```diff
@@ -0,0 +1,89 @@
+# Not Diamond
+
+When building complex LLM workflows users may need to prompt different models according to accuracy,
+cost, or call latency. Users can use [Not Diamond][nd] to route prompts in these workflows to the
+right model for their needs, helping maximize accuracy while saving on model costs.
+
+## Getting started
+
+Make sure you have [created an account][account] and [generated an API key][keys], then insert your API key below.
+
+![[Create an API key](imgs/notdiamond/api-keys.png)]
+
+## Tracing
+
+Weave integrates with [Not Diamond's Python library][python] to [automatically log API calls][ops].
+You only need to run `weave.init()` at the start of your workflow:
+
+```python
+from notdiamond import NotDiamond
+
+import weave
+weave.init()
+
+client = NotDiamond()
+session_id, provider = client.chat.completions.model_select(
+    messages=[
+        {"role": "system", "content": "You are a helpful assistant."},
+        {"role": "user", "content": "Concisely explain merge sort."}
+    ],
+    model=['openai/gpt-4o', 'anthropic/claude-3-5-sonnet-20240620']
+)
```

Comment:
@acompa 
It might be helpful to explain to the user what to do after this.

I have a `provider`, now what?